cmd/notation/internal/arg/arg.go

```diff
@@ -0,0 +1,17 @@
+package arg
+
+import (
+	"errors"
+	"fmt"
+)
+
+// ValidateCount validates the arguments are of expected length.
+func ValidateCount(args []string, expLen int, missingErrMsg string) error {
+	argsLength := len(args)
+	if argsLength < expLen {
+		return errors.New(missingErrMsg)
+	} else if argsLength > expLen {
+		return fmt.Errorf("expected %d argument(s) but found %d, arguments: %v", expLen, argsLength, args)
```

Comment:
The error message is inconsistent with `cobra`: see https://github.com/spf13/cobra/blob/v1.7.0/args.go#L94